go.mod

```diff
@@ -468,7 +468,7 @@ replace (
 	modernc.org/strutil => modernc.org/strutil v1.0.0
 	modernc.org/xc => modernc.org/xc v1.0.0
 	sigs.k8s.io/kustomize => sigs.k8s.io/kustomize v2.0.3+incompatible
-	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-e85c7b244fd2
+	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v0.0.0-20190302045857-ea680f03cc65b
```

Comment:
this isn't quite new enough. looks like `k8s.io/kube-openapi@v0.0.0-20190502190224-411b2483e503` requires `sigs.k8s.io/structured-merge-diff@v0.0.0-20190426204423-ea680f03cc65`